release/release-mac-arm

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+EXEDIR="$SCRIPT_DIR/.." 
+EXE=paintown
+APPVERSION=`git tag --sort=committerdate | grep -E '[0-9]' | tail -1 | cut -b 2-7`
+
+# inside tmp
+cd $SCRIPT_DIR
+rm -rf tmp && mkdir tmp && cd tmp
+
+# copy executable
+cp $EXEDIR/$EXE .
+
+echo "Before"
+otool -L $EXE
+
+# grab all shared libraries
+ALL_EXE_SHARED_LIBRARIES=`otool -L $EXE | awk '{print $1}' | grep opt | grep -v ':' | sed 's/^[[:space:]]*//'`
+
+# for each dep change from rpath to executable_path
+while IFS= read -r dylibpath; do
+
+    dylibpath_no_rpath=`echo $dylibpath | sed 's/@rpath\///'`
+    dylibname=`basename $dylibpath_no_rpath`
+
+    # copying $dylibname
+    cp $dylibpath_no_rpath .
+
+    # changing $dylibname
+    install_name_tool -change $dylibpath "@executable_path/../Frameworks/$dylibname" $EXE
+
+done <<< "$ALL_EXE_SHARED_LIBRARIES"
+
+echo "After"
+otool -L $EXE
+file $EXE
+
+# launcher
+echo '#!/bin/bash
+cd "`dirname "$0"`"
+
+if [ ! -d data ]; then
+    DATA_URL="https://github.com/kazzmir/paintown/releases/download/v3.6.0/data-3.6.0.zip"
```

Comment:
for now this is ok, but probably the data should be pulled from the paintown-data repo